packages/ui/src/components/va-stepper/VaStepper.demo.vue

```diff
@@ -172,6 +172,6 @@ const stepsWithNextAction = ref([
   { label: 'One' },
   { label: 'Two', beforeLeave: (step) => { step.hasError = true } },
   { label: 'Three', beforeLeave: (step) => model.value.c !== '' },
-  { label: 'Four', hasError: (step) => model.value.d === '' },
+  { label: 'Four', hasError: (step: any) => model.value.d === '' },
 ] as Step[])
```

Comment:
```suggestion
] satisfies Step[])
```

I think types in steps a bit messed-up. 

`hasError` might be a function, but only `boolean` is expected in Step type.

`beforeLeave` must allow return `boolean` or `void`